src/ResourceManager/DataMigration/AzureRM.DataMigration.Netcore.psd1

```diff
@@ -72,29 +72,38 @@ NestedModules = @('.\Microsoft.Azure.Commands.DataMigration.dll')
 FunctionsToExport = @()
 
 # Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
-CmdletsToExport = 'New-AzureRmDataMigrationDatabaseInfo', 
-    'New-AzureRmDataMigrationConnectionInfo', 
-    'New-AzureRmDataMigrationProject', 
-    'Remove-AzureRmDataMigrationProject', 
-    'New-AzureRmDataMigrationSqlServerSqlDbSelectedDB', 
-    'Get-AzureRmDataMigrationProject', 'Get-AzureRmDataMigrationService', 
-    'New-AzureRmDataMigrationService', 
-    'Remove-AzureRmDataMigrationService', 'New-AzureRmDataMigrationTask', 
-    'Get-AzureRmDataMigrationTask', 'Remove-AzureRmDataMigrationTask', 
-    'Start-AzureRmDataMigrationService', 
-    'Stop-AzureRmDataMigrationService', 'Stop-AzureRmDataMigrationTask'
+CmdletsToExport = 
+      'New-AzureRmDataMigrationDatabaseInfo', 
+      'New-AzureRmDataMigrationConnectionInfo', 
+      'New-AzureRmDataMigrationProject', 
+      'Remove-AzureRmDataMigrationProject', 
+      'New-AzureRmDataMigrationSqlServerSqlDbSelectedDB', 
+      'Get-AzureRmDataMigrationProject', 
+      'Get-AzureRmDataMigrationService', 
+      'New-AzureRmDataMigrationService', 
+      'Remove-AzureRmDataMigrationService', 
+      'New-AzureRmDataMigrationTask', 
+      'Get-AzureRmDataMigrationTask', 
+      'Remove-AzureRmDataMigrationTask', 
+      'Start-AzureRmDataMigrationService', 
+      'Stop-AzureRmDataMigrationService', 
+      'Stop-AzureRmDataMigrationTask',
+      'New-AzureRmDataMigrationSqlServerSqlDbMiSelectedDB',
```

Comment:
This list should be identical to the non-netcore psd1 - looks like they currently don't match, please update.